<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDiaryEdit</class>
 <widget class="QWidget" name="IDiaryEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="27">
      <widget class="QToolButton" name="toolExit">
       <property name="toolTip">
        <string>Exit</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QToolButton" name="toolPaste">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QToolButton" name="toolCopy">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="toolCut">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="toolRedo">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="toolUndo">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="13">
      <widget class="QToolButton" name="toolBold">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="14">
      <widget class="QToolButton" name="toolItalic">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="15">
      <widget class="QToolButton" name="toolUnder">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="16">
      <widget class="QToolButton" name="toolColor">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="18">
      <widget class="QComboBox" name="comboStyle"/>
     </item>
     <item row="0" column="17">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="22">
      <widget class="QToolButton" name="toolReload">
       <property name="toolTip">
        <string>Reload</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="23">
      <widget class="QToolButton" name="toolSave">
       <property name="toolTip">
        <string>Save</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="24">
      <widget class="QToolButton" name="toolPrint">
       <property name="toolTip">
        <string>Print</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="25">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="19">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkProfile">
       <property name="text">
        <string>show profiles</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkGeoCache">
       <property name="text">
        <string>show geocaches</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkAddMap">
       <property name="text">
        <string>add map view  when printing</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
